src/intl/messages.js

```diff
@@ -582,12 +605,16 @@ export const messages: { [messageId: string]: MessageType } = {
   sortOffFirst: 'Sorts off machines first',
   sortRunningFirst: 'Sorts running machines first',
   sortZToA: 'Sort from Z to A',
+  smartcard: 'Smartcard',
   smartcardEnabled: 'Smartcard enabled:',
+  smartcardTooltip: 'Smartcard is a physical card that can be used to authenticate to VMs connected to a SPICE console.',
   snapshot: 'Snapshots',
   snapshotRestore: 'Restore Snapshot',
   snapshotDelete: 'Delete Snapshot',
   snapshotInfo: 'All attached disks will be included in the snapshot. Content of memory can be included only if VM is running.',
   snapshotsTooltip: 'VM snapshots.',
+  spiceOptions: 'SPICE Options',
+  spiceProxy: 'SPICE Proxy',
```

Comment:
This is not required/planned as part of console options

Reply:
1. SPICE Options is a label for a section that groups all SPICE options (there are few of them)
2. SPICE Proxy is just in case here - it's present in the Web Admin UI

Reply:
Not all options on webadmin are relevant for web-ui.
It was already decided about those that are relevant as part of Laura's design, based on older user portal version and customer requirements.
I prefer not to add settings that we are not planning to implement :)